mlflow/lightgbm.py

```diff
@@ -379,6 +380,22 @@ def log_feature_importance_plot(features, importance, importance_type):
         else:
             kwargs["callbacks"] = [callback]
 
+        # We set free_raw_data to false on the Dataset object
+        # so that we can access the original data later.
+        train_data = args[1] if len(args) > 1 else kwargs["train_data"]
+
+        input_example = None
+        try:
+            if isinstance(train_data.data, str):
+                raise Exception("The input data was of type string.")
```

Comment:
should we use [MlflowException](https://github.com/mlflow/mlflow/blob/d743a40426d5dedbde395a4e6bbdeebadbccd4dc/mlflow/exceptions.py#L31) here?